vertical-pod-autoscaler/pkg/utils/metrics/recommender/recommender.go

```diff
@@ -48,7 +48,7 @@ var (
 			Namespace: metricsNamespace,
 			Name:      "vpa_objects_count",
 			Help:      "Number of VPA objects present in the cluster.",
-		}, []string{"update_mode", "has_recommendation", "api"},
+		}, []string{"update_mode", "has_recommendation", "api", "pods_matched"},
```

Comment:
nit: matches_pods is (for me) more suggestive of a bool value.

Reply:
Done.